apps/web/app/modules/Courses/Lesson/LessonContent.tsx

```diff
@@ -35,8 +36,11 @@ export const LessonContent = ({
   const [isNextDisabled, setIsNextDisabled] = useState(false);
   const { mutate: markLessonAsCompleted } = useMarkLessonAsCompleted();
   const { t } = useTranslation();
+  const { isAdminLike } = useUserRole();
 
   useEffect(() => {
+    if (isAdminLike) return;
+
```

Comment:
`React Hook useEffect has a missing dependency: 'isAdminLike'. Either include it or remove the dependency array`